ntbs-service/Authentication/DummyAuthHandler.cs

```diff
@@ -27,8 +27,11 @@ public class DummyAuthHandler : AuthenticationHandler<AuthenticationSchemeOption
             // Add role claim for base user role
             id.AddClaim(new Claim(ClaimTypes.Role, adfsOptions.AdGroupsPrefix + adfsOptions.BaseUserGroup, ClaimValueTypes.String));
             // Add role claim for user role - as specified in appsettings.Development.json
-            string group = adfsOptions.AdGroupsPrefix + (adfsOptions.DevGroup ?? adfsOptions.NationalTeamAdGroup);
-            id.AddClaim(new Claim(ClaimTypes.Role, group, ClaimValueTypes.String));
+            string groupAdmin = adfsOptions.AdGroupsPrefix + (adfsOptions.AdminUserGroup ?? adfsOptions.NationalTeamAdGroup);
+            id.AddClaim(new Claim(ClaimTypes.Role, groupAdmin, ClaimValueTypes.String));
```

Comment:
A bit too much copy-pasting here I think - using `adfsOptions.NationalTeamAdGroup in this instance doesn't really make sense?
I think you just want to have  
`id.AddClaim(new Claim(ClaimTypes.Role, adfsOptions.AdGroupsPrefix + adfsOptions.AdminUserGroup, ClaimValueTypes.String));`, and move this line above the comment in line 29..
Alternatively, we could change `adfsOptions.DevGroup` to `adfsOptions.UserGroups` and provide a list there, including the new admin group?